src/app/shared/components/displayable-list/displayableList.bindings.js

```diff
@@ -0,0 +1,5 @@
+export default {
+    displayableList: '<',
```

Comment:
I just realised, the component name and the binding name is the same? Change one or the other, otherwise will be messy

Reply:
Changed, but that wasnt the problem, still need the $timeout